rpm/centos-7/Dockerfile

```diff
@@ -18,6 +18,26 @@ ARG SUITE
 ENV DISTRO=${DISTRO}
 ENV SUITE=${SUITE}
 
+# Use recent Git version
+ARG GIT_VERSION="2.30.0"
+RUN yum install -y \
+    curl-devel \
+    expat-devel \
+    gettext-devel \
+    openssl-devel \
+    perl-CPAN \
+    perl-devel \
+    wget \
+    zlib-devel \
+  && yum --setopt=group_package_types=mandatory,default,optional groupinstall -y "Development Tools" \
+  && wget https://github.com/git/git/archive/v${GIT_VERSION}.tar.gz -O git.tar.gz \
+  && tar -zxf git.tar.gz \
+  && cd git-* \
+  && make prefix=/usr/local all \
+  && make prefix=/usr/local install \
+  && rm -rf /git* \
+  && git --version
```

Comment:
Ok needs to update Git for CentOS 7. I prefer to build from source as some third party repositories are not reliable and trusted. It takes a few minutes to build Git, so I think it's fine.